mail-client/mutt/mutt-1.6.1.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Small but very powerful text-based mail client"
+DESCRIPTION="A text-based email client for Unix-like systems. \
+It was originally written by Michael Elkins in 1995 and released \
+under the GNU General Public License version 2 or any later version.
+
+The Mutt slogan is All mail clients suck. This one just sucks less."
+HOMEPAGE="http://www.mutt.org"
+COPYRIGHT="1996-2014 Michael R. Elkins
+	1996-2002 Brandon Long
+	1997-2009 Thomas Roessler
+	1998-2005 Werner Koch
+	1999-2014 Brendan Cully
+	1999-2002 Tommi Komulainen
+	2000-2004 Edmund Grimley Evans
+	2006-2009 Rocco Rutte
+	2014-2015 Kevin J. McCarthy"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="ftp://ftp.mutt.org/pub/mutt/mutt-$portVersion.tar.gz"
+CHECKSUM_SHA256="98b26cecc6b1713082fc880344fa345c20bd7ded6459abe18c84429c7cf8ed20"
+PATCHES="mutt-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86 ?arm"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86 ?arm"
+
+PROVIDES="
+	mutt$secondaryArchSuffix = $portVersion
+	cmd:mutt$secondaryArchSuffix = $portVersion
+	cmd:flea$secondaryArchSuffix = $portVersion
+	cmd:muttbug$secondaryArchSuffix = $portVersion
+	cmd:pgpewrap$secondaryArchSuffix = $portVersion
+	cmd:pgpring$secondaryArchSuffix = $portVersion
+	cmd:smime_keys$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgpgme$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libgpgme$secondaryArchSuffix
+	lib:libgpg_error$secondaryArch
+	lib:libgpg_error.so.0$secondaryArch
```

Comment:
I think you should replace
```
	lib:libgpg_error$secondaryArch
	lib:libgpg_error.so.0$secondaryArch
```
by just
```
	lib:libgpg_error$secondaryArch
```
(if you don't need a specific version of the lib),
or by
```
	lib:libgpg_error$secondaryArch >= 0
```
(if you need libgpg_error.so.0.)